packages/RichTextEditing-Tests.package/RichTextTextStructureTests.class/instance/testLinkTextStructureCanNotDeriveFromPrettyPrint.st

```diff
@@ -0,0 +1,4 @@
+unit-tests
+testLinkTextStructureCanNotDeriveFromPrettyPrint
+
+	self deny: (self linkTextStructure couldDeriveFromPrettyPrinting)
```

Comment:
Is this important or just to increase coverage?

Reply:
the latter